pkg/volume/portworx/portworx_util.go

```diff
@@ -181,13 +186,24 @@ func createDriverClient(hostname string) (*osdclient.Client, error) {
 	}
 }
 
-func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost) (volumeapi.VolumeDriver, error) {
+func (util *PortworxVolumeUtil) getPortworxDriver(volumeHost volume.VolumeHost, localOnly bool) (volumeapi.VolumeDriver, error) {
```

Comment:
Feel free to link to some external docs if the context is large enough to fit in a code comment. We frequently do that in AWS and other volume plugins (or we should be doing that).